.travis.yml

```diff
@@ -45,6 +45,10 @@ matrix:
     fast_finish: true
     include:
 
+        # Try with pint
+        - python: 3.6
```

Comment:
For 3.4 etc. I found that the majority of failing tests in this version have nothing to do with the library but with the way condo handles dependencies. They often fail because numpy is at an older version and all that depend on it also can't upgrade, so tests fail. But it's quite difficult to keep track of it.
If you have a strong motivation to explicitly support 3.4, we can surely test it on 3.4 too.
3.6 is more crucial because it tests for all new library versions, so it anticipates the problems that users will have, while 3.4 is more for backwards compatibility (which I see as less critical at this stage of development).